src/game/assets/faction/sheet/Sheet.module.css

```
@@ -22,14 +22,22 @@
+  --faction-sheet-page-aspect: 141.428571%;
+  --faction-sheet-page-rule: 0.3vw;
+}
+
 .page {
   position: relative;
   background-color: #fffdf1;
-  /* A4 portrait aspect ratio (297 / 210). */
-  padding-top: 141.428571%;
+  padding-top: var(--faction-sheet-page-aspect);
 
   font-size: 3vw;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)